src/Language/English/EnglishDictionary.php

```diff
@@ -66,6 +66,7 @@ class EnglishDictionary implements Dictionary
         'LVL' => [['lat'], ['sentim']],
         'LYD' => [['dinar'], ['cent']],
         'MAD' => [['dirham'], ['cent']],
+        'AED' => [['Dirham'], ['Fils']],
```

Comment:
Could you please add the currency in the alphabetical order (top of the array).

Reply:
this is done now thx